.circleci/documentation-versions.py

```diff
@@ -1,16 +1,43 @@
 """
-Version helper.
+Version helper updates file version-helper.js
+which takes care of injecting versions into rtd template.
+After version-helper.js is updated,
+script prints paths of old versions to be removed.
 
-Checks the directory 'build_dir' and return list of last 7 versions.
-[dev, tag, tag-1, tag-2,..., tag-5]
+EXAMPLE
+
+docs directory (sceptre.github.io) contains following directories
+
+    ├── 1.5.0
+    ├── 2.1.3
+    ├── 2.1.4
+    ├── 2.1.5
+    ├── 2.2.0
+    ├── 2.2.1
+    ├── dev
+    └── latest -> 2.2.1 (symlink)
+
+Script settings:
+KEEP_SPECIFIC_VERSIONS = ["1.5.0"]
+KEEP_ACTIVE_VERSIONS = 3
+
+The content of version-helper.js will be:
+    ['latest', 'dev', '2.2.1', '2.2.0', '2.1.5', '1.5.0']
+    (where latest points to 2.2.1)
+Output of versions to remove will be:
+    <path>/2.1.4
+    <path>/2.1.3
 
-Then updates file version-helper.js, which takes care of injecting versions
-into rtd template.
-Return paths, for old versions to be removed.
 """
 import os
 from operator import attrgetter
 import sys
+import re
+
+VERSION_RE = re.compile(r"\d+.\d+.\d+")
+KEEP_SPECIFIC_VERSIONS = ["1.5.0"]  # these versions won't be removed
+# number of active versions to keep, eg. len([2.2.1, 2.2.0, 2.1.9]) == 3
+KEEP_ACTIVE_VERSIONS = 3
```

Comment:
If you rename to something like `NUMBER_OF_VERSIONS_TO_KEEP` you could lose the comment above. 

Also just checking this should be 7?

Reply:
The number does not need to be 7 anymore.
Reason for 7 was that the whole list contained 7 items `len(['latest', 'dev', v2.1,..vN..])==7`.
We do not count `dev`, `latest` and `len(KEEP_SPECIFIC_VERSIONS)`  towards the total number of active versions.

Reply:
the output of `versions_to_remove` is used in 
https://github.com/Sceptre/sceptre/blob/master/.circleci/github-pages.sh#L72

I wanted to avoid deleting files in python since it was part of a shell script anyway.